validator/client/validator.go

```diff
@@ -931,6 +944,87 @@ func (v *validator) logDuties(slot types.Slot, duties []*ethpb.DutiesResponse_Du
 	}
 }
 
+// PrepareBeaconProposer calls the prepareBeaconProposer RPC to set the proposer information such as fee recipient.
+func (v *validator) PrepareBeaconProposer(ctx context.Context, km keymanager.IKeymanager) error {
+	if km == nil {
+		return errors.New("keymanager is nil when calling PrepareBeaconProposer")
+	}
+	if v.prepareBeaconProposalConfig == nil {
```

Comment:
This shouldnt happen right?

Reply:
this shouldn't happen correct. I will remove for now